frontend/components/templates/QuestionPageLayout.tsx

```diff
@@ -133,6 +133,9 @@ const QuestionsPageLayout = ({
                                 view_count={question.views_count}
                                 tags={question.tags}
                                 user={question.user}
+                                team_name={
+                                    question.team?.name === undefined ? '' : question.team?.name
```

Comment:
I think you can use this. It returns the right-hand value if its left-hand value is null/undefined.

```suggestion
                                    question.team?.name ?? ' '
```